tests/suites/test_suite_hmac_drbg.function

```diff
@@ -12,8 +12,9 @@ static int mbedtls_test_entropy_func( void *data, unsigned char *buf, size_t len
 {
     entropy_ctx *ctx = (entropy_ctx *) data;
 
-    if( len > ctx->len )
+    if( len > ctx->len ) {
```

Comment:
Style: adding braces is not necessary and not preferred. If there is a brace, it must go on the next line.